mappings/net/minecraft/block/entity/EnderChestBlockEntity.mapping

```diff
@@ -3,6 +3,6 @@ CLASS net/minecraft/class_2611 net/minecraft/block/entity/EnderChestBlockEntity
 	FIELD field_12005 ticks I
 	FIELD field_12006 viewerCount I
 	FIELD field_12007 animationProgress F
-	METHOD method_11218 canPlayerUse (Lnet/minecraft/class_1657;)Z
+	METHOD method_11218 isAccessible (Lnet/minecraft/class_1657;)Z
```

Comment:
I think the original name was clearer.

Reply:
Original name is kind of clunky, imo.

Reply:
Why not just canUse or can Access? Only a player would be using it anyway.

Reply:
I wanted to avoid the `canX` naming, as I feel that should be reserved for checking the capabilities of _this_ object.

Think of `player.canUse(tool)`

Reply:
Inventories don't really use players, they get used _by_ players, so the terminology is kind of backwards...

Reply:
`isOpenable`?

Reply:
`canBeOpened`?

Reply:
This is not like an initial open condition but like if the player can keep opening it

Reply:
hm ok isAccessible is ok. Perhaps `isAccessibleBy`?

Reply:
> This is not like an initial open condition but like if the player can keep opening it
If that's the case, maybe I should call it `canStayOpen`?